.github/workflows/ci.yml

```diff
@@ -13,9 +13,15 @@ on:
   schedule:
     - cron: '0 9 * * *'
   push:
-    branches: [master]
+    branches:
+      - master
+      - 'release-.*'
+    tags:
+      - 'v[1-9][0-9]*.[0-9]+.[0-9]+'
```

Comment:
can you use the release tag regex from TravisCI?

`v([1-9][0-9]*!)?(0|[1-9][0-9]*)(\.([0-9]*))*((a|b|rc)(0|[1-9][0-9]*))?(\.post(0|[1-9][0-9]*))?(\.dev(0|[1-9][0-9]*))?`

Based on my reading of https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet it should be compatible